modules/pipeline/services/pipelinesvc/create_task.go

```diff
@@ -183,14 +182,30 @@ func (s *PipelineSvc) calculateTaskRunAfter(action *pipelineyml.Action) []string
 }
 
 // judgeTaskExecutor judge task executor by action info
-func (s *PipelineSvc) judgeTaskExecutor(action *pipelineyml.Action, actionSpec *apistructs.ActionSpec) (spec.PipelineTaskExecutorKind, spec.PipelineTaskExecutorName) {
+func (s *PipelineSvc) judgeTaskExecutor(task *spec.PipelineTask, actionSpec *apistructs.ActionSpec) (spec.PipelineTaskExecutorKind, spec.PipelineTaskExecutorName) {
```

Comment:
add ut

Reply:
already existed in pipelinesvc.create_task_test.go file